### PR TITLE
updated 2019 roadmap

### LINKED ISSUE #399: Update the roadmap for 2019
**Motivation**
The roadmap needs to be updated to reflect the current direction of the project

**Scope of change**
ROADMAP.md only

### PR DESCRIPTION
**Pull request by Soloflex95**
Fixes #399 .

Changes in this pull request:
- Removed Batch Change heading
- Added new roadmap items to 2019

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -6,35 +6,24 @@ we anticipate the "plan" to change.  This document will be updated regularly to
...
-1. If the user is an admin (VinylDNS or Zone admin), they can make any changes to DNS records
-1. If the user is granted access to make the change by an ACL rule, the change is allowed
-1. Otherwise, the change is rejected with a `Not Authorized`
+1. RFC is available [here](https://github.com/vinyldns/rfcs/pull/6)
 
-The final phase of work will add several new features to Batch Changes.  Those include:
+## Reverse DNS Lookup Records (PTR)
+**Planned for: Q1 2019**
+DNS records are commonly used to determine the IP addresses associated to Fully Qualified Domain Names(FQDN). Another common
+use is to allow a lookup in the opposite direction.  In order to accomplish this, you neet a PTR record type.
```
Comment (nimaeskandary) [inline ROADMAP.md]: neet -> need, and maybe add a sentence "A combined `A/PTR` record type would optimize this workflow

Comment (pauljamescleary) [reply ROADMAP.md]: we support PTR records already.  We shouldn't have a section dedicated to PTR.  The A+PTR and AAAA+PTR are part of batch change.  We should leave in batch changes

File: ROADMAP.md [hunk 2]
```diff
@@ -52,6 +41,9 @@ There are several other features that we would like to support.  We will be open
 1. DNS SEC - There is no first-class support for DNS SEC.  That feature set is being defined.
 1. Record meta data - VinylDNS will allow the "tagging" of DNS records with arbitrary key-value pairs
 1. DNS Global Service Load Balancing (GSLB) - Support for common DNS GSLB use cases and integration with various GSLB vendors
+1. Profanity filter - reject changes that use profane language in record labels or text.
```
Comment (pauljamescleary) [inline ROADMAP.md]: Why are these pulled down into Other?  Blacklisting as it was envisioned is now being implemented as high-value domains.  Could pull that up into Q1 under batch change.  Blacklisting by itself the way it was split off from high value domains should probably go away.

**Comment by britneywright** [comment]
Closing this for now. Roadmap will be updated further after more planning.

**Pull request closed without merging**